app/utils/crypto/encryption_algorithm.rb

```diff
@@ -0,0 +1,28 @@
+# frozen_string_literal: true
+
+class Crypto::EncryptionAlgorithm
```

Comment:
can't we just add this to Crypto::Symmetric ?
does it make sense to have it's own class here?